integration-test/experimental-react/package.json

```diff
@@ -4,9 +4,8 @@
   "version": "0.0.0",
   "type": "module",
   "scripts": {
-    "prepare": "rm -rf node_modules/@apollo/client-react-streaming; mkdir node_modules/@apollo/client-react-streaming && cp -r ../../packages/client-react-streaming/{package.json,dist} node_modules/@apollo/client-react-streaming",
```

Comment:
Did this change intentionally? I saw your other comment about making sure that we don't have multiple React versions installed, but maybe you fixed that in this PR?

Reply:
This is just a moving target 🤣 

At some point, the `prepare` script was necessary for this, then I changed it to an `exec:` approach (and removed this script), which works nicely in CI, but then it turned out that rebuilding locally becomes really annoying, so there will again be a similar script in the next PR... 🙃